### PR TITLE
gh/workflows: tune LVH VM params

### PR DESCRIPTION
**Pull request by brb**
Successful run - https://github.com/cilium/cilium/actions/runs/3580705455.

### CODE REVIEW

**Review by nbusseneau — CHANGES_REQUESTED** [review]
Needs rebase following merge of #22423. There will also be a conflict with #22440, but if you handle the upgrade in this PR then #22440 should automatically get closed.

**Review by nbusseneau — APPROVED** [review]

File: .github/workflows/conformance-datapath.yaml [hunk 1]
```diff
@@ -186,21 +186,14 @@ jobs:
...
         with:
           test-name: datapath-conformance
-          image: kind
-          lvh-version: 'v0.0.1'
           image-version: ${{ matrix.kernel }}
           host-mount: ./
           cmd: |
-            set -eux
-
-            echo 'nameserver 1.1.1.1' > /etc/resolv.conf
```
Comment (pchaigno) [inline .github/workflows/conformance-datapath.yaml:199]: The commit description doesn't explain why this isn't needed anymore.

**Review by pchaigno — CHANGES_REQUESTED** [review]

File: .github/workflows/conformance-datapath.yaml (hunk 1 above)
Comment (brb) [reply .github/workflows/conformance-datapath.yaml:199]: Good catch, added to the commit msg.

**Review by pchaigno — APPROVED** [review]

**Comment by joestringer** [comment]
Conflicts/regressions prevented automatic backport of this PR. @brb , if you wish this to land in v1.13 please provide the backport.

**Comment by joestringer** [comment]
Never mind, this landed already in RC4. I don't know why the labels indicated it wasn't backported yet. I've updated them.

**Pull request merged**